tests/test_preprints.py

```diff
@@ -256,6 +256,13 @@ class TestOnPreprintUpdatedTask(OsfTestCase):
     def setUp(self):
         super(TestOnPreprintUpdatedTask, self).setUp()
         self.user = AuthUserFactory()
+        if len(self.user.fullname.split(' ')) > 2:
```

Comment:
Is this still necessary, given the way this PR changes how fullname is generated?